DISCOVER/04_venue_selection.md

```diff
@@ -101,7 +106,9 @@
 **Further Reading:**
 
 - [How to accommodate a breastpumping mom at your event](http://miriamposner.com/blog/how-to-accommodate-a-breastpumping-mom-at-your-event/)
-- [How to Set Up a Lactation Room to Support Breastfeeding at Work](https://www.thespruce.com/lactation-room-breastfeeding-at-work-3545106)
+
+<!-- NOT WORKING
+- [How to Set Up a Lactation Room to Support Breastfeeding at Work](https://www.thespruce.com/lactation-room-breastfeeding-at-work-3545106)-->
```

Comment:
found the page at: https://web.archive.org/web/20190717082756/https://www.verywellfamily.com/lactation-room-breastfeeding-at-work-3545106